variables.tf

```diff
@@ -336,6 +339,12 @@ variable "runner_gitlab_registration_token_secure_parameter_store_name" {
   default     = "gitlab-runner-registration-token"
 }
 
+variable "runner_gitlab_access_token_secure_parameter_store_name" {
```

Comment:
suggestion: the variables were grouped with the v7 release. The prefix `runner_gitlab` already indicates that the variable should be moved to the `runner_gitlab` block and the prefix removed.